scripts/trivia.coffee

```diff
@@ -39,39 +64,79 @@ module.exports = (robot) ->
           msg.send solutionHint
       , 15000
 
-  #function to turn off trivia
-  #made this so i could reset every time a question is done
-  triviaStop = (msg) ->
-    clearInterval(hintInterval)
-    hintInterval = null
-    triviaOn = false
-    
-  #solve attempt matches anything after !, case insensitive
-  #need to add some way to score points
-  robot.respond /!(.+)$/i, (msg) ->
-    solveAttempt = msg.match[1]
+  #-------------Solve Attempt-----------------#
+  #solve attempt matches anything after ?  
+  robot.respond /\?\s*(.+)$/i, (msg) ->
+    solveAttempt = msg.match[1].trim()
```

Comment:
And you should really check if trivia is running before you do any of this stuff. Because as of right now if you do "? ajsdkflajd" he responds with incorrect even if trivia is not running